website/server/controllers/api-v3/members.js

```diff
@@ -57,6 +58,41 @@ api.getMember = {
   },
 };
 
+/**
+ * @api {get} /api/v3/members/:memberId/achievements Get member achievements object
```

Comment:
Sure! Wasn't done previously because the exact stuff the API would return hadn't been decided yet.